filament/src/details/Camera.cpp

```diff
@@ -204,7 +219,7 @@ void FCamera::setExposure(float aperture, float shutterSpeed, float sensitivity)
 }
 
 double FCamera::getFocalLength() const noexcept {
-    return (FCamera::SENSOR_SIZE * mProjection[1][1]) * 0.5;
+    return (FCamera::SENSOR_SIZE * mEyeProjection[0][1][1]) * 0.5;
```

Comment:
this deserves a comment and I would use a local variable for projection[0]. Probably need to update the documentation of getFocalLength.